generator-scrivito/app/index.js

```diff
@@ -0,0 +1,172 @@
+const Generator = require("yeoman-generator");
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+    this.log("Welcome in scrivito Obj/Widgets generator!");
+    this.option("obj", { type: String });
+    this.option("widget", { type: String });
```

Comment:
Please remove both options. The code should be called with `yo scrivito:widget` and *not*  with `yo scrivito --widget`.